src/nvim/tui/tui.c

```diff
@@ -54,12 +54,8 @@
 #define TOO_MANY_EVENTS 1000000
 #define STARTS_WITH(str, prefix) (strlen(str) >= (sizeof(prefix) - 1) \
     && 0 == memcmp((str), (prefix), sizeof(prefix) - 1))
```

Comment:
```
  src/nvim/tui/tui.c:56:  Inner expression should be aligned as opening brace + 1 (+ 2 in case of {). Relevant opening is on line 55  [whitespace/alignment] [2]
```

Reply:
Done.